L1Trigger/L1TMuonEndCap/src/TrackFinder.cc

```diff
@@ -3,136 +3,22 @@
-                                          minBX,
-                                          maxBX,
-                                          bxWindow,
-                                          bxShiftCSC,
-                                          bxShiftRPC,
-                                          bxShiftGEM,
-                                          era_,
-                                          zoneBoundaries,
-                                          zoneOverlap,
-                                          includeNeighbor,
-                                          duplicateTheta,
-                                          fixZonePhi,
-                                          useNewZones,
-                                          fixME11Edges,
-                                          pattDefinitions,
-                                          symPattDefinitions,
-                                          useSymPatterns,
-                                          thetaWindow,
-                                          thetaWindowZone0,
-                                          useRPC_,
-                                          useSingleHits,
-                                          bugSt2PhDiff,
-                                          bugME11Dupes,
-                                          bugAmbigThetaWin,
-                                          twoStationSameBX,
-                                          maxRoadsPerZone,
-                                          maxTracks,
-                                          useSecondEarliest,
-                                          bugSameSectorPt0,
-                                          readPtLUTFile,
-                                          fixMode15HighPt,
-                                          bug9BitDPhi,
-                                          bugMode7CLCT,
-                                          bugNegPt,
-                                          bugGMTPhi,
-                                          promoteMode7,
-                                          modeQualVer);
-    }
-  }
-
-}  // End constructor: TrackFinder::TrackFinder()
+      tokenDTPhi_(iConsumes.consumes<emtf::DTTag::digi_collection>(iConfig.getParameter<edm::InputTag>("DTPhiInput"))),
+      tokenDTTheta_(
+          iConsumes.consumes<emtf::DTTag::theta_digi_collection>(iConfig.getParameter<edm::InputTag>("DTThetaInput"))),
+      tokenCSC_(iConsumes.consumes<emtf::CSCTag::digi_collection>(iConfig.getParameter<edm::InputTag>("CSCInput"))),
+      tokenCSCComparator_(iConsumes.consumes<emtf::CSCTag::comparator_digi_collection>(
+          iConfig.getParameter<edm::InputTag>("CSCComparatorInput"))),
+      tokenRPC_(iConsumes.consumes<emtf::RPCTag::digi_collection>(iConfig.getParameter<edm::InputTag>("RPCInput"))),
+      tokenRPCRecHit_(
+          iConsumes.consumes<emtf::RPCTag::rechit_collection>(iConfig.getParameter<edm::InputTag>("RPCRecHitInput"))),
```

Comment:
Seems that the `tokenRPCRecHit_` is not used so maybe a simple fix would be to remove this parameter and `consumes()` call?